cmd/res/configuration.toml

```diff
@@ -0,0 +1,62 @@
+[Writable]
+LogLevel = 'INFO'
+
+[Service]
+BootTimeout = 30000
+CheckInterval = '10s'
+Host = '0.0.0.0'
+ServerBindAddr = ''  # blank value defaults to Service.Host value
+Port = 49960
```

Comment:
Device service WG need to assign the default port to avoid collisions.

Reply:
+1  It was changed to 49950.  Assuming @iain-anderson coordinated on the port.  If so, this is fixed.

Reply:
Assigned port 49994, please update